doc/topics/development/contributing.rst

```diff
@@ -100,7 +100,8 @@ Fork a Repo Guide_>`_ and is well worth reading.
     .. note::
 
         If your change fixes a bug or implements a feature already filed in the
-        `issue tracker <GitHub issue tracker>`_, be sure to reference the issue
+        `issue tracker <https://guides.github.com/features/issues/>`_, be sure to
```

Comment:
The URL to the Github issue tracker is at the bottom of the document. It should point to salt's issue, not the github guide... as it does at the bottom.

Reply:
Ah, I see. Yeah I wasn't understanding what they wanted there. I'll make the link adjustment.

Reply:
@alan-cugler ping.